File: permissions/src/main/java/com/gabriel/permissions/ui/controller/response/AuthenticationResponse.java
package com.gabriel.permissions.ui.controller.response;

import com.fasterxml.jackson.annotation.JsonProperty;

public record AuthenticationResponse(
    @JsonProperty("access_token") String accessToken,
    @JsonProperty("expires_in") Integer expiresIn,
    @JsonProperty("token_type") String tokenType) {
}

File: permissions/src/main/kotlin/com/gabriel/permissions/ui/controller/response/AuthenticationResponse.kt
package com.gabriel.permissions.ui.controller.response

import com.fasterxml.jackson.annotation.JsonProperty

data class AuthenticationResponse(
    @JsonProperty("access_token") val accessToken: String,
    @JsonProperty("expires_in") val expiresIn: Int,
    @JsonProperty("token_type") val tokenType: String
)
